node/test/lib/alloc-cluster.js

```diff
@@ -109,14 +109,22 @@ function allocCluster(opts) {
 
     function createChannel(i) {
         var chan = TChannel(extend(channelOptions));
-        var port = opts.listen && opts.listen[i] || 0;
-        chan.on('listening', chanReady);
-        chan.listen(port, host);
         cluster.channels[i] = chan;
 
+        var port = opts.listen ? opts.listen[i] : 0;
+        if (port === null) {
```

Comment:
Should document somewhere that null is a sentinel value that means “do not listen on any port”.